### ISSUE TITLE
Hue Haunted House not animating

### ISSUE THREAD

**Issue by ohmantics** [diyhue/diyHue#1059]
### Issue does not already exist?

No, I did not search

### Select Environment

Docker

### Home Assistant related?

No

### Description

Installed DIYHue as a HomeAssistant add-on on an HA Yellow.
Added a Hue bulb to HA directly via ZHA.
Connected DIYHue to HA with a token.
Added the bulb to a room via Hue Essentials.
Run Hue Haunted House app on iOS.
Connect to DIYHue bridge via virtual link button.
Enable the one bulb in the Lights tab.
Turn on an animation like Hellfire.
Bulb does not animate at all.
DIYHue is logging tons of PUTs to transitiontime on that light.

### Errorlog:

[diyhue.log](https://github.com/user-attachments/files/18138731/diyhue.log)

### Steps to reproduce

Installed DIYHue as a HomeAssistant add-on on an HA Yellow.
Added a Hue bulb to HA directly via ZHA.
Connected DIYHue to HA with a token.
Added the bulb to a room via Hue Essentials.
Run Hue Haunted House app on iOS.
Connect to DIYHue bridge via virtual link button.
Enable the one bulb in the Lights tab.
Turn on an animation like Hellfire.
Bulb does not animate at all.
DIYHue is logging tons of PUTs to transitiontime on that light.

### Please enter your operating system details here

Hue-Emulator Version: 2024-12-14 15:43:21.000000000
WebUI Version: 2024-12-14 15:43:29.000000000
Architecture: aarch64
OS: Linux
Linux version: #1 SMP PREEMPT Tue Dec 3 19:23:20 UTC 2024
Linux release: 6.6.51-haos-raspi
Hardware: HomeAssistant Yellow with CM4

### Upload debug tar

[config_debug.tar.zip](https://github.com/user-attachments/files/18138735/config_debug.tar.zip)

### What DiyHue version(branch) are you using?

master (latest)

**Comment by hendriksen-mark** [comment]
Hi, unfortunately his app is a paid app so we can not test this.
In the logs you provide i can see that this app uses V1API.
It wants to make a group with type `LightGroup` this is not in diyhue responses.
That is why you get the http code 500.
```
2024-12-14 19:01:49,370 - flaskUI.restful - 154 - INFO - {'type': 'LightGroup', 'name': 'HauntedHouseG', 'lights': ['39']}
2024-12-14 19:01:49,374 - werkzeug - 225 - INFO - 192.168.0.45 - - [14/Dec/2024 19:01:49] "[35m[1mPOST /api/e510bf40ba9011ef9b4ee45f01bbd792/groups HTTP/1.1[0m" 500 -
```
in this app can you select other rooms or is the room page empty?
Can you share some screenshots from the app?

**Comment by ohmantics** [comment]
Not sure if it's turning up in this log, but this app creates those groups based on your light selections. Perhaps that's failing?
![IMG_2156](https://github.com/user-attachments/assets/4f7720a0-c172-47e7-a09d-abbf91795a8d)
![IMG_2155](https://github.com/user-attachments/assets/c3eccc02-b4a6-40e3-9e56-ece272d61878)

The same author makes a whole bunch of occasion-specific apps that share the same engine. I'd be happy to send a donation on Ko-Fi or elsewhere to cover having at least one for your testing. The whole point for me in adopting DIYHue is to rid myself of the Hue Bridges I have and unwind all the complex VLAN and firewall nonsense I had to do to prevent them from updating to recent cloud-required spyware versions. Supporting this project is a no-brainer.

**Comment by hendriksen-mark** [comment]
if you make a donation to ko-fi the ends up with the mastermind of diyhue(Marius Motea).
Marius transfers the money to me if it is designed for me.

**Comment by ohmantics** [comment]
OK, running through Proxyman, I see a POST to /groups: `{"type":"LightGroup","name":"HauntedHouseG","lights":["39"]}`
and that's returning 500. It superficially looks correct in the Hue v1 API docs.

**Comment by hendriksen-mark** [comment]
Yes I know, that is why I asked you if any other groups are displayed in the app, to add `LightGroup` we have to add allot(group 0 is also called lightgroup).
I have been testing with this.

**Comment by hendriksen-mark** [comment]
Hi, thank you for your donation.
I have installed the haunted app and it works flawless with my setup.
I can select my lights without problems.
The lights change color or brightness without errors.
Color and non color lights.
I dont see any option to make rooms in the haunted app.
All my rooms I made in the hue app appear in the haunted app
I suggest you make rooms in the hue app for all of your lights and then select the lights again in the haunted app.

**Comment by ohmantics** [comment]
The app uses LightGroups to sync its animations among multiple lights. It only uses your Rooms as part of its Light selection UI, not during animation.

I look forward to the next release with your changes. Thanks!

**Comment by hendriksen-mark** [comment]
Please try the new version and give feedback.

**Comment by ohmantics** [comment]
It's still not animating for me, but I'm not seeing errors now.
[diyhue.log](https://github.com/user-attachments/files/18247069/diyhue.log)

**Comment by hendriksen-mark** [comment]
And if you look in the log directly of the container? The traceback is not logged to the .log file.

**Comment by hendriksen-mark** [comment]
https://diyhue.readthedocs.io/en/latest/AddFuncts/debug.html
